src/logic/calculate.js

```diff
@@ -0,0 +1,120 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
```

Comment:
- [ ] I recommend that you add a default `zero` in the screen when you press `AC` and consider Adding a default zero in the screen when you just opened the app

Reply:
Thanks for pointing this out. I have done the necessary changes

Reply:
This issue is well addressed! :+1: :clap: